### ISSUE TITLE
Import wallet contract problem

### ISSUE THREAD

**Issue by Maximus078** [ethereum/mist#4032]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.11.1`
OS & Version: windows and osx
Node version: `geth 1.8.11`
Number of blocks synchronized: 6033110
```

Trying to import wallet contract (multisig). 
WALLETS -> ADD WALLET CONTRACT -> IMPORT WALLET, fill the address, owner, name and click "Create". Nothing happens. Type of sync (fast/light/full), OS (WIN/OS X) does not matter, ETHEREUM WALLET and MIST behave the same way. 

meanwhile 10.0 does not have this problem.
![image](https://user-images.githubusercontent.com/41737677/43259925-92a89204-90e0-11e8-92c8-174d8d2ee92b.png)

**Comment by ChristBKK** [comment]
hey,
i have the same problem

**Comment by marcgarreau** [comment]
thanks for reporting it. we'll look to get that resolved for 0.12.

**Comment by ChristBKK** [comment]
thanks @marcgarreau    can you estimate when you want to release the new version?  Do you aim to release monthly?

**Comment by ryanio** [comment]
Hey everyone thanks for your bug reports.

I found some contract calls that weren't upgraded properly which is why importing wasn't working. I just fixed them in https://github.com/ethereum/meteor-dapp-wallet/pull/487/commits/78993979a311044d31a3fe878cafc5c884dd4332 so it should be fixed in the next release, likely in the next few weeks.

Best,
Ryan

**Comment by fredbuluku** [comment]
That’s in order thanks 🙏

**Comment by narodnik** [comment]
Same problem. Unable to import wallet. The create button doesn't do anything.

**Comment by rash8** [comment]
Same problem. At the transition to "sync mode - full" and at the subsequent restart ethereum wallet, all my contract wallets have disappeared.

Now I can not add them. Please solve as soon as possible the problem.

**Comment by dannylee71** [comment]
Wallet id:b4cb75be-8248-4d31-8397-7d291ce4e9ca

**Comment by neonoxGER** [comment]
Is there any workaround? I need to import my wallet.

**Comment by graph1994** [comment]
Same issue, do you guys need help wrapping this fix up?

**Comment by ChristBKK** [comment]
@ryanio   can you guys release this soon?  already over 4 weeks since the latest version which is bugged.

**Comment by photojacketman** [comment]
What is the workaround and any idea when will 0.12.0 be released?

**Comment by graph1994** [comment]
Since I needed access to my tokens, I just reinstalled an older version and it worked 🤣

**Comment by photojacketman** [comment]
There's a catch 22 for me here, the older version wont fully sync, despite having a SAMSUNG 860 EVO (550Mb/s) and there is no light option natively in the GUI.

**Comment by graph1994** [comment]
Yeah I had a similar issue awhile ago, I had to jump start geth with light(since the UI was borked), go into the terminal and try running geth `geth --syncmode="light"`

**Comment by photojacketman** [comment]
Yes, I know this one well also, however no matter which version I use (even 0.10.0), it just does this and nothing else:

![image](https://user-images.githubusercontent.com/30982282/46249138-a1f73680-c424-11e8-86c0-c602b430c327.png)

I'm using this version of Ethereum Wallet. 

![image](https://user-images.githubusercontent.com/30982282/46249149-d3700200-c424-11e8-8e41-f87bb051389e.png)


By the way I launch geth.exe before launching the wallet.

**Comment by photojacketman** [comment]
Last note :)

I'm using this version of Geth, which one are you using?

![image](https://user-images.githubusercontent.com/30982282/46249315-4b3f2c00-c427-11e8-8922-2d6f7fd7cecc.png)

**Comment by ironboy1** [comment]
Is there any hope, that we will see a fix for this still in this year?
3.5 month since the last (really buggy) release….

**Comment by jambtt** [comment]
Try closing EW then deleting the Ethereum Wallet folder under AppData\Roaming and restart EW try importing your multi-sig then.

**Comment by jussiry** [comment]
I was able to retrieve my ETH from wallet contract by first syncing geth from command line (fastsync, not sure if light would have been enough), and then starting old version of Ethereum Wallet (0.8.9) with geth still running on background. With fully synced geth, importing wallet contract worked like a charm.

Good luck to anyone having this problem! It's not fun having your ETH locked up and not being sure if they are still retrievable.

**Comment by jonpspitz** [comment]
Still nothing? Same issue. Been a headache. Would love an update on if there will actually be an update.
